info/buildSrc/src/main/kotlin/org/cqfn/diktat/generation/docs/GenerationDocs.kt

```diff
@@ -131,6 +133,27 @@ enum class FindType {
     ;
 }
 
+private fun handleHyperlinks(line: String): String {
+    var correctedString = line
+    if (correctedString.contains(HYPERLINKS)) {
+        val hyperlinkSubString = HYPERLINKS.findAll(correctedString)
+        hyperlinkSubString.forEach {
+            var (text, link) = HYPERLINKS.find(it.value)!!.destructured
+            // need to replace back, because it is a hyperlink
+            link = link.trim().drop(1).dropLast(1) // drop ( and )
+            link = link.replace("\\#", "#")
+            link = link.replace("\\&", "&")
+            link = link.replace("\\_", "_")
+            // need to replace ` in hyperlink, because it breaks latex compilation
+            text = text.replace("`", "")
```

Comment:
I think you still need to remove `[]` from `text` here: now it looks like `\href{https://docs.oracle.com/en/Kotlin/Kotlinse/11/tools/KDoc.html}{[Official Document]}` in tex. Are these brackets rendered too? If yes, they are not needed in the rendered result, link should be just highlighted.

Reply:
Fixed